managed-services/data-lab/concepts.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: Distributed Data Lab - Concepts
+  description: Learn the fundamental concepts of Scaleway Distributed Data Lab.
+content:
+  h1: Distributed Data Lab - Concepts
+  paragraph: Learn the fundamental concepts of Scaleway Distributed Data Lab.
+tags: distributed data lab apache spark notebook jupyter processing
+dates:
+  validation: 2024-07-31
+categories:
+  - managed-services
+---
+
+## Data Lab
+
+A Data Lab is a project setup that combines a Notebook and an Apache Spark Cluster for data analysis and experimentation. it comes with the required infrastructure and tools to allow data scientists, analysts, and researchers to explore data, create models, and gain insights.
+
+## Distributed Data Lab
+
+A Distributed Data Lab is a data lab that is distributed across multiple worker nodes to accelerate the processing of large datasets to save time and gain access to actionable insights faster.
+
+## Fixture
+
+A fixture is a set of data forming a request used for testing purposes.
+
+## JupyterLab 
+
+JupyterLab is a web-based platform for interactive computing, letting you work with notebooks, code, and data all in one place. It builds on the classic Jupyter Notebook by offering a more flexible and integrated user interface, making it easier to handle various file formats and interactive components.
+
+## Kubernetes Cluster
```

Comment:
question: as it's how it works internally (not exposed to the client), do we want to share it with the user?